### PR TITLE
xen: Add recipe for xen version 4.19

### PR DESCRIPTION
**Pull request by dsemenets**
Recipe for xen 4.19.
Depend on meta-xt-common layer will be removed with this PR.

### CODE REVIEW

File: recipes-extended/xen/files/xen-config.cfg [hunk 1]
```diff
@@ -1,3 +1,6 @@
+CONFIG_XSM=y
+CONFIG_IPMMU_VMSA=y
```
Comment (lorc) [inline recipes-extended/xen/files/xen-config.cfg:2]: Why do you need RCAR IPMMU driver here?

File: recipes-extended/xen/xen_4.19.bb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```
Comment (lorc) [inline recipes-extended/xen/xen_4.19.bb:1]: Is there are reason why you are not using recipe provided by meta-xt-common? You commit message does not tell anything about this

Comment (oleksiimoisieiev) [reply recipes-extended/xen/xen_4.19.bb:1]: I'm also don't understand why you decided to remove changes you proposed in one of the already merged prs/

Comment (dsemenets) [reply recipes-extended/xen/xen_4.19.bb:1]: @lorc Yes
@oleksiimoisieiev This version doesn't depend on meta-xt-common. It means we don't have to wait until meta-xt-common will move to scarthgap

File: recipes-extended/xen/files/xen-config.cfg (hunk 1 above)
Comment (dsemenets) [reply recipes-extended/xen/files/xen-config.cfg:2]: I'll delete this

**Pull request closed without merging**